sites/all/modules/custom/dccr_blocks/dccr_blocks.module

```diff
@@ -369,3 +387,278 @@ function dccr_blocks_create_account_block() {
 
   return $block;
 }
+
+/**
+ * Get Final Schedule block.
+ */
+function dccr_get_final_schedule($only_flagged, $empty_slot_text, $block_title) {
+  $block = array();
+  $markup = '';
+  $terms = dccr_get_rooms();
+  $tracks = taxonomy_get_tree(4);
+  $terms_count = 0;
+  $counter = 0;
+  $params = drupal_get_query_parameters();
+  $markup .= '<div class=\'tracks\'>';
+  for ($i = 0; $i < count($tracks); $i++) {
+    $markup .= '<div class=\'color-' . strtolower($tracks[$i]->name) . '\'><p> ' . $tracks[$i]->name . '</p></div>';
+  }
+  $markup .= '</div><div class=\'filter\'><div class=\'day\'>' . l(t('July 30th'), 'schedule', array('query' => array('date' => 'thursday'))) . '</div><div class=\'day\'>' . l(t('July 31th'), 'schedule', array('query' => array('date' => 'friday'))) . '</div></div>';
+
+  if ($only_flagged) {
+    drupal_add_js(drupal_get_path('module', 'dccr_blocks') . '/js/alter_flag_behavior.js');
+  }
+
+  // Thursday, July 30th.
+  $date1 = 1438264800;
```

Comment:
could you set this values as constans at the beginning of the file and use the constant in the rest of the file/site? Please